package.json

```diff
@@ -22,7 +22,15 @@
     "babel-preset-stage-2": "6.24.1",
     "banner-footer-webpack-plugin": "git://github.com/dtothefp/banner-footer-webpack-plugin.git",
     "browser-sync": "2.18.7",
-    "capital-framework": "3.6.1",
+    "cf-buttons": "4.4.1",
+    "cf-core": "4.3.1",
+    "cf-expandables": "4.1.3",
+    "cf-forms": "4.3.2",
+    "cf-grid": "4.2.1",
+    "cf-icons": "4.1.1",
+    "cf-layout": "4.4.1",
+    "cf-pagination": "4.1.3",
+    "cf-typography": "4.1.2",
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3092/commits/c8e229829bf7428dc0075c4a05d416ad41b75c85